app/views/comments/_form.html.erb

```diff
@@ -1,20 +1,27 @@
-<%= simple_form_for [@article, @article.comments.build] do |form| %>
-  <div>
-    <%= form.input :commenter %>
-  </div>
-  <br>
-  <div>
-    <%= form.input :body %>
-  </div>
-  <br>
-  <div>
-    <%= form.input :status, label: t('comment.form.status'),
-                   collection: {
-                     t('article.form.statuses.pub') => 'published',
-                     t('article.form.statuses.hidden') => 'hidden',
-                     t('article.form.statuses.archived') => 'archived',
-                   },
-                   selected: :public %>
-  </div>
-  <%= form.button :submit %><br>
-<% end %>
+<div class="container">
+  <%= bootstrap_form_for [@article, @article.comments.build] do |form| %>
```

Comment:
It's better to use one type of forms. You don't really need `bootstrap_form_for`. Look at articles form, I provided you an example. You can update here and delete this gem at all.